README.md

```diff
@@ -1,16 +1,17 @@
 # README
 
-**1. Backlog**  
+## Chess App Workflow
+
+*1. Backlog*  
 * Choose task from backlog
-* "git pull origin master" to pull down latest code 
-* (rake db:drop, create, migrate, and seed locally)-- ***need to update***
-* Create branch in local environment from "master"
-* 'git checkout -b NAME_OF_BRANCH'
-* Push to Github 'git push origin NAME_OF_BRANCH'
+* "git pull origin master" to pull down latest code
+* ****bundle exec rake db: locally to drop, create, migrate, & seed local DB***
+* Create branch in local environment from "master" (never from an existing branch) ---> ‘git checkout -b NAME_OF_BRANCH’
```

Comment:
This steps ensures that the new branch is a clean copy from the master branch. If someone updates the master branch while you are working on your local branch, then while you are on your branch, checkout to the master branch, git pull origin master, checkout to your branch, and git merge master from your branch

e.g.,
git pull origin master
git checkout -b  my_branch
....<---someone updates master branch--->
git checkout master
git pull origin master
git checkout my_branch
git merge master

If there are any merge conflicts, then resolve them before pushing your branch up to git hub